app/components/forms/orders/attendee-list.js

```diff
@@ -25,25 +29,39 @@ export default class AttendeeList extends Component {
   }
 
   @action
-  downloadTicketForAttendee(eventName, orderId, attendeeId) {
-    this.loader.downloadFile(`/orders/attendees/${attendeeId}.pdf`)
-      .then(res => {
-        const anchor = document.createElement('a');
-        anchor.style.display = 'none';
-        anchor.href = URL.createObjectURL(new Blob([res], { type: 'application/pdf' }));
-        anchor.download = `${eventName}-Ticket-${orderId}-${attendeeId}.pdf`;
-        document.body.appendChild(anchor);
-        anchor.click();
-        this.notify.success(this.l10n.t('Here is your ticket'),
-          {
-            id: 'tick_pdf'
-          });
-        document.body.removeChild(anchor);
-      })
-      .catch(e => {
-        console.error('Error while downloading tickets', e);
-      })
-      .finally(() => {
-      });
+  saveHolder(holder) {
+    holder.save();
+    this.data.save();
+  }
+
+  @action
+  async downloadTicketForAttendee(eventName, orderId, attendeeId) {
+    try {
+      if (!this.ticketDowloaded) {
+        await this.confirm.prompt(this.l10n.t('Please check the filled detail carefully. Once you Download ticket, the information can\'t be changed on it.'));
```

Comment:
I didn't get this prompt while I downloaded the ticket. :thinking:  Other things worked though.